### PR TITLE
Fixed get_bulk_threads associated contest

### PR DESCRIPTION
**Pull request by kurtisassad**
Due to model change, the query for the contest in get_bulk_threads also needs to change. I am assuming here that contests are related to contestActions through (contest_address, contest_id). If this is not correct, please let me know in the comments.

## Link to Issue
Closes: #8060

## Description of Changes
- Add extra condition on contest_address

## Test Plan
Run:
```sql
INSERT INTO "ContestManagers" (contest_address, community_id, interval, created_at, name, image_url, funding_token_address, prize_percentage, payout_structure, ticker, decimals)
VALUES ('contest_address_1', 'dydx', 30, NOW(), 'Contest 1', 'https://example.com/image1.jpg', 'funding_token_address_1', 10, '{1, 2, 3}', 'ticker_1', 18);


INSERT INTO "Contests" (contest_address, contest_id, start_time, end_time)
VALUES ('contest_address_1', 1, NOW() - INTERVAL '1 day', NOW() + INTERVAL '1 day'),
('contest_address_1', 2, NOW() - INTERVAL '2 day', NOW() + INTERVAL '2 day');

INSERT INTO "ContestActions" (contest_address, contest_id, content_id, actor_address, action, content_url, voting_power, created_at, thread_id)
VALUES ('contest_address_1', 1, 1, 'actor_address_1', 'upvoted', 'https://example.com/content1', 10, NOW(), 11167),
('contest_address_1', 1, 1, 'actor_address_2', 'upvoted', 'https://example.com/content1', 10, NOW(), 11167);
```

Then check the query:
```sql
SELECT
              TT.id as thread_id,
              json_agg(json_strip_nulls(json_build_object(
              'id', CON.contest_id,
              'thread_id', TT.id,
              'content_id', CA.content_id,
              'start_time', CON.start_time,
              'end_time', CON.end_time
          ))) as "associatedContests"
          FROM "Contests" CON
          JOIN "ContestActions" CA ON CON.contest_id = CA.contest_id AND CON.contest_address = CA.contest_address
          JOIN "Threads" TT ON TT.id = CA.thread_id
          GROUP BY TT.id
```

ensure this returns the thread with the correct data.

### CODE REVIEW

File: packages/commonwealth/server/controllers/server_threads_methods/get_bulk_threads.ts [hunk 1]
```diff
@@ -264,7 +268,7 @@ export async function __getBulkThreads(
     }
   `,
     {
-      //logging: true,
+      logging: true,
```
Comment (masvelio) [inline packages/commonwealth/server/controllers/server_threads_methods/get_bulk_threads.ts:271]: should still be true?

**Review by masvelio — APPROVED** [review]
works as expected!


https://github.com/hicommonwealth/commonwealth/assets/14819225/d15e0cd6-1876-4ae9-8535-0f5860829274

File: packages/commonwealth/server/controllers/server_threads_methods/get_bulk_threads.ts (hunk 1 above)
Comment (kurtisassad) [reply packages/commonwealth/server/controllers/server_threads_methods/get_bulk_threads.ts:271]: Good catch, will remove

**Review by rbennettcw — APPROVED** [review]
LGTM

**Pull request merged**